skimage/measure/_structural_similarity.py

```diff
@@ -3,36 +3,137 @@
 __all__ = ['structural_similarity']
 
 import numpy as np
+from scipy.ndimage.filters import uniform_filter, convolve1d
 
 from ..util.dtype import dtype_range
-from ..util.shape import view_as_windows
 
 
-def structural_similarity(X, Y, win_size=7,
-                          gradient=False, dynamic_range=None):
+def gaussian_filter2(X, sigma=1.5, size=11):
+    """ nD Gaussian filter with specific window extent.
+
+    matches the implementation of Wang. et. al.
+
+    Parameters
+    ----------
+    X : ndarray
+        image
+    sigma : float
+        Gaussian standard deviation (pixels)
+    size : float
+        Gaussian kernel extent (pixels)
+
+    Returns
+    -------
+    X : ndarray
+        filtered image
+
+    Notes
+    -----
+    scipy.ndimage.gaussian is very similar, but uses a 13 tap FIR filter
+    rather than the 11 tap one of Wang. et. al.
```

Comment:
In practice it may not be a big deal to just use the 13-tap filter and making a brief note about the difference.  I think the result should agree to within 2-3 decimal places either way, which is typically the level of precision that ssim values are tabulated in the literature.

**Here are some concrete numbers for a noisy vs. clean cameraman image:**
*Matlab code of Wang et. al.*:
`0.327314295673357`
*Gaussian with window size 11, gaussian_weights=True, use_sample_covariance=False*
`0.32731429567337322`   (identical to >10 decimal places)
*window size 13, gaussian_weights=True, use_sample_covariance = False*
`0.32785353312812454`   (first few decimals the same, probably good enough)
*old sckit image v0.11 implementation with the constant window of size 7*
`0.34192589699605191`   (already differs at the second decimal)